### PR TITLE
docs: added information on building for locale with npm

### PR DESCRIPTION
**Pull request by SandersAaronD**
Added information on building for locale with npm. Grunt still works fine but the documentation didn't cover the correct way to pass arguments to grunt through npm. Since this is core npm behavior, it isn't something that can or should be changed, but should be documented so that it can be used for building.

Closes issue: #2649

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,10 @@ Axe can be built using your local language. To do so, a localization file must b
 
 `grunt build --lang=nl`
 
+or equivalently:
+
+`npm build -- --lang=nl`
```
Comment (straker) [inline README.md:97]: This seems to fail for me, but adding `run` seems to fix it.

```suggestion
`npm run build -- --lang=nl`
```

**Review by straker — CHANGES_REQUESTED** [review]
Thanks for the doc change. Just a minor fix

File: README.md (hunk 1 above)
Comment (straker) [reply README.md:97]: Better yet, we should probably not promote the use of the `grunt` step and just use the `npm run build` process for this.

Comment (SandersAaronD) [reply README.md]: "There should be one-- and preferably only one --obvious way to do it."

**Review by straker — APPROVED** [review]

**Comment by straker** [comment]
Approved security and pr title.

**Pull request merged**